integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageTest.java

```diff
@@ -464,6 +621,143 @@ public void testTreeNodeRefresh() throws InterruptedException {
+    // 7. click fileset tree node
+    String filesetNode =
+        String.format(
+            "{{%s}}{{%s}}{{%s}}{{%s}}{{%s}}",
+            METALAKE_NAME,
+            FILESET_CATALOG_NAME,
+            CATALOG_TYPE_FILESET,
+            SCHEMA_NAME_FILESET,
+            FILESET_NAME);
+    catalogsPage.clickTreeNode(filesetNode);
+    // 8. verify show tab details
+    Assertions.assertTrue(catalogsPage.verifyShowDetailsContent());
+    Assertions.assertTrue(
+        catalogsPage.verifyShowPropertiesItemInList(
+            "key", PROPERTIES_KEY1, PROPERTIES_KEY1, false));
+    Assertions.assertTrue(
+        catalogsPage.verifyShowPropertiesItemInList(
+            "value", PROPERTIES_KEY1, PROPERTIES_VALUE1, false));
+  }
+
+  @Test
+  @Order(21)
+  public void testViewKafkaCatalog() throws InterruptedException {
+    // 1. back to the list catalog of metalake
+    catalogsPage.metalakeSelectChange(METALAKE_SELECT_NAME);
+    Assertions.assertTrue(catalogsPage.verifyEmptyCatalog());
+
+    catalogsPage.metalakeSelectChange(METALAKE_NAME);
+    // 2. create topic of kafka catalog
+    createTopic(METALAKE_NAME, KAFKA_CATALOG_NAME, SCHEMA_NAME, TOPIC_NAME);
+    // 3. click kafka catalog tree node
+    String kafkaCatalogNode =
+        String.format(
+            "{{%s}}{{%s}}{{%s}}", METALAKE_NAME, KAFKA_CATALOG_NAME, CATALOG_TYPE_MESSAGING);
+    catalogsPage.clickTreeNode(kafkaCatalogNode);
+    // 4. verify show table title、 schema name and tree node
+    Assertions.assertTrue(catalogsPage.verifyShowTableTitle(CATALOG_TABLE_TITLE));
+    Assertions.assertTrue(catalogsPage.verifyShowDataItemInList(SCHEMA_NAME, false));
+    List<String> treeNodes =
+        Arrays.asList(
+            MODIFIED_HIVE_CATALOG_NAME,
+            ICEBERG_CATALOG_NAME,
+            MYSQL_CATALOG_NAME,
+            PG_CATALOG_NAME,
+            FILESET_CATALOG_NAME,
+            KAFKA_CATALOG_NAME,
+            SCHEMA_NAME);
+    Assertions.assertTrue(catalogsPage.verifyTreeNodes(treeNodes));
+    // 5. click schema tree node
+    String kafkaSchemaNode =
+        String.format(
+            "{{%s}}{{%s}}{{%s}}{{%s}}",
+            METALAKE_NAME, KAFKA_CATALOG_NAME, CATALOG_TYPE_MESSAGING, SCHEMA_NAME);
+    catalogsPage.clickTreeNode(kafkaSchemaNode);
+    // 6. verify show table title、 fileset name and tree node
+    Assertions.assertTrue(catalogsPage.verifyShowTableTitle(SCHEMA_TOPIC_TITLE));
+    Assertions.assertTrue(catalogsPage.verifyShowDataItemInList(TOPIC_NAME, false));
+    treeNodes =
+        Arrays.asList(
+            MODIFIED_HIVE_CATALOG_NAME,
+            ICEBERG_CATALOG_NAME,
+            MYSQL_CATALOG_NAME,
+            PG_CATALOG_NAME,
+            FILESET_CATALOG_NAME,
+            KAFKA_CATALOG_NAME,
+            SCHEMA_NAME,
+            TOPIC_NAME);
+    Assertions.assertTrue(catalogsPage.verifyTreeNodes(treeNodes));
+    // 7. click fileset tree node
+    String filesetNode =
```

Comment:
The variable name should be `kafkaNode`?

Reply:
It should be topicNode, I have changed it correct.